packages/dev-server-rollup/test/node/plugins/commonjs.test.ts

```diff
@@ -116,20 +116,19 @@ exports.default = _default;`;
       const text = await fetchText(`${host}/foo.js`);
       expectIncludes(
         text,
-        "import * as commonjsHelpers from '/__web-dev-server__/rollup/commonjsHelpers.js?web-dev-server-rollup-null-byte=%00commonjsHelpers.js';",
+        '/__web-dev-server__/rollup/commonjsHelpers.js?web-dev-server-rollup-null-byte=%00commonjsHelpers.js',
       );
-      expectIncludes(
-        text,
-        'var foo = commonjsHelpers.createCommonjsModule(function (module, exports) {',
-      );
-      expectIncludes(text, 'exports.__esModule = true;');
-      expectIncludes(text, 'exports.default = void 0;');
+      // expectIncludes(
+      //   text,
```

Comment:
Is this intentional? Then lets remove it.

Reply:
jup intentional... left in so you can see that we will no longer test this implementation detail... not sure if it's an important implementation details...

if not then we can remove it

Reply:
Yeah wr test impl details, but then we do know its doing its job. So we can remove this then